services/core-web/src/components/modalContent/ViewNoticeOfDepartureModal.js

```diff
@@ -292,8 +292,8 @@ let ViewNoticeOfDepartureModal = (props) => {
         </Form.Item>
         <Row justify="space-between" className="padding-md--top" gutter={24}>
           <Col span={12}>
-            <p className="field-title">Technical Operations Director</p>
-            <p className="content--light-grey padding-md">{EMPTY_FIELD}</p>
+            <p className="field-title">Updated Date</p>
```

Comment:
Was there a design change here?  I don't see any change on Figma.

Reply:
No, but there is no actual use of those dummy fields so decided to remove them for now.